assets/helpers/utilities.js

```diff
@@ -76,6 +76,12 @@ function deserialiseJsonObject(serialised: string): ?Object {
 
 }
 
+// Copied from https://github.com/playframework/playframework/blob/38abd1ca6d17237950c82b1483057c5c39929cb4/framework/src/play/src/main/scala/play/api/data/validation/Validation.scala#L80
```

Comment:
Is this the same validation regex they're using on the endpoint? It seems to allow dotless email addresses, e.g. `cp.scott@gu` as opposed to `cp.scott@gu.com`, which also throw the contributions error.